java/test/jmri/configurexml/LoadAndStoreTestBase.java

```diff
@@ -196,6 +196,24 @@ public static void checkFile(File inFile1, File inFile2) throws Exception {
                 }
             }
 
+            // window y positions will sometimes differ based on window decorations.
+            if (!match) {
+                if (line1.contains("  <LayoutEditor") ||
+                    line1.contains(" <switchboardeditor")) {
+                    // if either line contains a y position attribute
+                    String yposition_regexe = "( y=\"[^\"]*\")";
+                    String[] splits1 = line1.split(yposition_regexe);
+                    if (splits1.length == 2) {  // (yes) remove it
+                        line1 = splits1[0] + splits1[1];
+                    }
+                    String[] splits2 = line2.split(yposition_regexe);
+                    if (splits2.length == 2) {  // (yes) remove it
+                        line2 = splits2[0] + splits2[1];
+                    }
```

Comment:
Please do this for X as well as Y. When I run this test on my laptop with Debian Linux, I consistent get wrong X values on some items. I don't know why, but by adding your code, but for the X axis, removes the problem.

In the tests that failes, x=0 is expected, but I get x=-5.

Reply:
sure, thanks for the suggestion.  I did a little refactoring at the same time.

Reply:
Thanks. The static analysis failed since line 197 and 198 of LoadAndStoreTestBase should have `xposition_regexe` instead of `yposition_regexe`.

Reply:
Yeah, I saw that.  should be fixed now.